### PR TITLE
chore: Add support for polish language

### LINKED ISSUE #2567: Update Open Event Attendee App To Support more Languages
**Describe the feature you'd like**
Many people may not understand all basic languages and thus may need the app translated to their native language. But as of now the app does not seem to offer any such option. I believe the app should eventually at some time have the ability to support atleast all major foreign languages apart from English.

**Would you like to work on the issue?**
- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by aryanguls**
Fixes #2567 #2514 

Changes: For the purpose of this merge I have added support for polish language to the Open Event Attendee Android App to further increase the number of languages it supports.

### CODE REVIEW

File: app/src/main/res/values-pol/string.xml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    <string name="sign_in_canceled">logowanie zostało anulowane</string>
+    <string name="cannot_fetch_location">nie można pobrać lokalizacji/string>
+
+    <!--welcome fragment-->
+    <string name="start_text_1">rozpocznij tekst 1</string>
+    <string name="start_text_2">rozpocznij tekst 2</string>
+    <string name="pick_a_city">wybierz miasto</string>
+
+    <!--sign up fragment-->
+    <string name="invalid_email_message">niepoprawna wiadomość e-mail/string>
```
Comment (nikit19) [inline app/src/main/res/values-pol/string.xml]: You need to close this tag.

**Review by nikit19 — CHANGES_REQUESTED** [review]
Before sending a pull request, please check if the project is building successfully or not in your computer.

**Comment by aryanguls** [comment]
@nikit19 Sir I have resolved the changes you mentioned and have also checked the entire files for other such errors. Please review and merge.

**Comment by aryanguls** [comment]
@nikit19 @hpdang @mariobehling Mentors please review and accept.

**Review by nikit19 — APPROVED** [review]

**Pull request merged**